src/styles/global.css

```diff
@@ -15,51 +15,10 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-/* Inter unexpectedly contains various codepoints which collide with emoji, even
-   when variation-16 is applied to request the emoji variant.  From eyeballing
-   the emoji picker, these are: 20e3, 23cf, 24c2, 25a0-25c1, 2665, 2764, 2b06, 2b1c.
-   Therefore we define a unicode-range to load which excludes the glyphs
-   (to avoid having to maintain a fork of Inter). */
 :root {
-  /* This is a false positive */
-  /* stylelint-disable unit-no-unknown */
-  --cpd-inter-unicode-range: U+0000-20e2, U+20e4-23ce, U+23d0-24c1, U+24c3-259f,
```

Comment:
I just don't know about that. I know the @font-face in @fontsource/inter has some unicode ranges, but I don't know if it properly excludes the emojis or not. Do we have a way to test that?

Reply:
So i've actually removed `global.css` from the exported CSS. 
It's still used in Storybook, but right now it is not a great idea to have all of that.

This bit is particularly critical for Element Web, but a bit less for other projects. We do have visual regression tests for it on Percy yes. But only for EW.